docs/general/administration/configuration.md

```diff
@@ -115,6 +115,25 @@ Jellyfin uses fonts to render text in many places.
 
 The system fonts installed on the server are used for burning in subtitles and rendering cover images. How to install them depends on the operating system.
 
+### Server Fonts in Docker

```

Comment:
We should specify that it is server side system fonts for docker, since server fonts could also mean "fallback fonts"